package.json

```diff
@@ -84,111 +84,111 @@
   "homepage": "https://www.brave.com/",
   "dependencies": {
     "acorn": "3.2.0",
-    "ad-block": "^3.0.5",
+    "ad-block": "~3.0.5",
     "aphrodite": "1.1.0",
-    "async": "^2.0.1",
-    "bat-balance": "^1.0.1",
-    "bat-client": "^1.0.5",
-    "bat-publisher": "^1.0.2",
-    "bignumber.js": "^4.0.4",
+    "async": "~2.5.0",
+    "bat-balance": "~1.0.1",
+    "bat-client": "~1.0.5",
+    "bat-publisher": "~1.0.2",
+    "bignumber.js": "~4.0.4",
     "bloodhound-js": "brave/bloodhound",
-    "clipboard-copy": "^1.0.0",
-    "compare-versions": "^3.0.1",
-    "electron-localshortcut": "^0.6.0",
+    "clipboard-copy": "~1.2.0",
+    "compare-versions": "~3.1.0",
+    "electron-localshortcut": "~0.6.1",
     "electron-squirrel-startup": "brave/electron-squirrel-startup",
-    "emoji-regex": "^6.5.1",
+    "emoji-regex": "~6.5.1",
     "file-loader": "0.11.2",
-    "font-awesome": "^4.5.0",
+    "font-awesome": "~4.7.0",
     "font-awesome-webpack": "0.0.4",
-    "fs-extra": "^2.1.2",
-    "immutable": "^3.7.5",
-    "immutablediff": "^0.4.2",
+    "fs-extra": "~2.1.2",
+    "immutable": "~3.8.1",
+    "immutablediff": "~0.4.4",
     "immutablepatch": "brave/immutable-js-patch",
-    "l20n": "^3.5.1",
-    "lru-cache": "^1.0.0",
+    "l20n": "~3.5.1",
+    "lru-cache": "~1.1.1",
     "moment": "~2.18.1",
-    "niceware": "^1.0.4",
-    "parse-torrent": "^5.8.1",
-    "prettier-bytes": "^1.0.3",
-    "prop-types": "^15.5.6",
-    "punycode": "^2.0.0",
+    "niceware": "~1.0.4",
+    "parse-torrent": "~5.8.3",
+    "prettier-bytes": "~1.0.4",
+    "prop-types": "~15.6.0",
+    "punycode": "~2.1.0",
     "qr-image": "3.2.0",
     "random-lib": "2.1.0",
-    "react": "~15.6.1",
-    "react-dnd": "^2.1.4",
-    "react-dnd-html5-backend": "^2.1.2",
-    "react-dom": "^15.5.4",
-    "string.prototype.endswith": "^0.2.0",
-    "string.prototype.startswith": "^0.2.0",
+    "react": "~15.6.2",
+    "react-dnd": "~2.5.1",
+    "react-dnd-html5-backend": "~2.5.1",
+    "react-dom": "~15.6.2",
+    "string.prototype.endswith": "~0.2.0",
+    "string.prototype.startswith": "~0.2.0",
     "tablesort": "5.0.1",
     "tldjs": "1.6.2",
-    "tracking-protection": "1.1.x",
+    "tracking-protection": "~1.1.6",
```

Comment:
`tracking-protection@1.1.6` on https://gist.github.com/luixxiul/1cf8689da05e509d45176588730e5a16#file-npm-txt-L2738. To apply the same range specification.